updating/updating-restricted-network-cluster.adoc

```diff
@@ -55,6 +55,17 @@ include::modules/update-restricted.adoc[leveloffset=+1]
 
 include::modules/images-configuration-registry-mirror.adoc[leveloffset=+1]
 
+[id="updating-rhel-nodes-restricted-network"]
+== Updating RHEL nodes on a restricted network
+
+Before you can update {op-system-first} nodes, you must mirror your repository and obtain the {product-title} packages.
+
+include::modules/installation-mirror-repository.adoc[leveloffset=+2]
```

Comment:
RHEL worker nodes are a part of the whole cluster, user MUST firstly upgrade the rhcos cluster firstly, then upgrade rhel worker. In https://deploy-preview-33007--osdocs.netlify.app/openshift-enterprise/latest/updating/updating-restricted-network-cluster.html#update-mirror-repository_updating-restricted-network-cluster, user already did that when upgrading rhcos cluster, so I think it is redundant to be linked here again when upgrading rhel workers.